test/unit/discovered_extensions_test.rb

```diff
@@ -4,6 +4,8 @@ class FindDiscoveryRulesTest < ActiveSupport::TestCase
   include Foreman::Controller::DiscoveredExtensions
 
   setup do
+    SETTINGS[:organizations_enabled] = true
```

Comment:
I think I found it. One test turned off organizations, looks like some tests are being scheduled after this one.